### ISSUE TITLE
TypeError: value.replace is not a function

### ISSUE THREAD

**Issue by goliatone** [michaelsogos/pg-diff#10]
```
$ pg-diff -c development initial-migration
                              _   _    __    __                  _   _ 
  _ __     __ _            __| | (_)  / _|  / _|           ___  | | (_)
 | '_ \   / _` |  _____   / _` | | | | |_  | |_   _____   / __| | | | |
 | |_) | | (_| | |_____| | (_| | | | |  _| |  _| |_____| | (__  | | | |
 | .__/   \__, |          \__,_| |_| |_|   |_|            \___| |_| |_|
 |_|      |___/                                                        

     Author: [object Object]
    Version: 1.2.8
 PostgreSQL: 9.6+
    License: MIT
Description: PostgreSQL schema and data comparing tool


CONFIGURED OPTIONS
         Script Author: @G
      Output Directory: .../ops/migrations/sqlscripts
     Schema Namespaces: public
     Idempotent Script: ENABLED
          Data Compare: ENABLED


Connected to PostgreSQL 11.4 (Debian 11.4-1.pgdg90+1) on [192.168.99.100:5432/dev] ✓
Connected to PostgreSQL 11.4 (Debian 11.4-1.pgdg90+1) on [192.168.99.100:6543/pro] ✓

Collect SOURCE database objects
[|||||||||||||||||||] 100% - Database objects collected!

Collect TARGET database objects
[|||||||||||||||||||] 100% - Database objects collected!

Compare SOURCE with TARGET database objects
[|||||||||||||||||||] 100% - Database objects compared!

Collect SOURCE tables records
[|||||||||||||||||||] 100% - Table records collected!

Collect TARGET tables records
[|||||||||||||||||||] 100% - Table records collected!

Compare SOURCE with TARGET database table records
[-------------------] 0% - Comparing table "public"."entity" records

TypeError: value.replace is not a function
TypeError: value.replace is not a function
    at Object.__generateSqlFormattedValue (.../ops/migrations/node_modules/pg-diff-cli/src/sqlScriptGenerator.js:353:34)
    at Object.generateInsertTableRecordScript (.../ops/migrations/node_modules/pg-diff-cli/src/sqlScriptGenerator.js:316:35)
    at sourceTableRecords.records.rows.forEach (.../ops/migrations/node_modules/pg-diff-cli/src/compareRecords.js:100:45)
    at Array.forEach (<anonymous>)
    at Object.__compareTableRecords (.../ops/migrations/node_modules/pg-diff-cli/src/compareRecords.js:69:41)
    at Object.compareTablesRecords (.../ops/migrations/node_modules/pg-diff-cli/src/compareRecords.js:44:22)
    at __runComparison (.../ops/migrations/node_modules/pg-diff-cli/main.js:306:49)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by goliatone** [comment]
It looks like the field triggering this issue is of type `json` but seems to be defined as:
```js
Field {
    name: 'metadata',
    tableID: 38850,
    columnID: 13,
    dataTypeID: 114,
    dataTypeSize: -1,
    dataTypeModifier: -1,
    format: 'text' }
```

**Comment by michaelsogos** [comment]
@goliatone

Thanks, I will check the issue. 
To replicate can you supply me the table schema and records? A dbeaver backup will be perfect.

**Comment by michaelsogos** [comment]
> @goliatone
> Thanks, I will check the issue.
> To replicate can you supply me the table schema and records? A dbeaver backup will be perfect.

Hi @goliatone
Are you allowed to share you table schema and records?

**Comment by goliatone** [comment]
@michaelsogos here is the table:

```sql
CREATE TABLE device (
    id text PRIMARY KEY,
    "assetTag" text UNIQUE,
    "deviceId" text,
    name text,
    description text,
    status text,
    coordinates json,
    metadata json,
    "createdAt" timestamp with time zone,
    "updatedAt" timestamp with time zone
);
```

**Comment by michaelsogos** [comment]
Dear @goliatone 

I released version 1.2.9 that introduce support for **JSON** and **XML** as by your needs 😄 .

**Comment by goliatone** [comment]
@michaelsogos thank you! will test it out and let you know
